tst/teststandard/processes/children.tst

```diff
@@ -0,0 +1,18 @@
+gap> d := DirectoryCurrent();;
+gap> scriptdir := DirectoriesLibrary( "tst/teststandard/processes/" );;
+gap> checkpl := Filename(scriptdir, "check.pl");;
+
+# If IO is loaded, disable it's signal handler
+gap> if IsBoundGlobal("IO_RestoreSIGCHLDHandler") then
+> ValueGlobal("IO_RestoreSIGCHLDHandler")();
```

Comment:
I assume that's just for the current `io` version, until https://github.com/gap-packages/io/pull/55 is merged?

Reply:
Yes, if you comment this out, the test fails (it "should" fail, the manual explictly says that GAP's process stuff can't be used if IO's signal handler is enabled, it just got more likely to trigger).